lib/ohai/plugins/linux/hostnamectl.rb

```diff
@@ -26,8 +26,8 @@
     hostnamectl_path = which("hostnamectl")
     if hostnamectl_path
       shell_out(hostnamectl_path).stdout.split("\n").each do |line|
-        key, val = line.split(":")
-        hostnamectl[key.chomp.lstrip.tr(" ", "_").downcase] = val.chomp.lstrip
+        key, val = line.split(": ")
```

Comment:
This would fail if e.g., I have my location set to something like `AWS: us-west-1b`. This prevents that:

```suggestion
        key, val = line.split(": ", 2)
```